### ISSUE TITLE
CvpClient.logout() does not work

### ISSUE THREAD

**Issue by freedge** [aristanetworks/cvprac#131]
CvpClient.logout() seems not working at all (CVP 2020.2.3, cvprac 1.0.4)

```
Traceback (most recent call last):
  File "upgrade.py", line 7, in <module>
    clnt.logout()
  File "/home/centos/myenv/lib64/python3.6/site-packages/cvprac/cvp_client.py", line 554, in logout
    response = self.session.post('/login/logout.do')
  File "/home/centos/myenv/lib64/python3.6/site-packages/requests/sessions.py", line 590, in post
    return self.request('POST', url, data=data, json=json, **kwargs)
  File "/home/centos/myenv/lib64/python3.6/site-packages/requests/sessions.py", line 528, in request
    prep = self.prepare_request(req)
  File "/home/centos/myenv/lib64/python3.6/site-packages/requests/sessions.py", line 466, in prepare_request
    hooks=merge_hooks(request.hooks, self.hooks),
  File "/home/centos/myenv/lib64/python3.6/site-packages/requests/models.py", line 316, in prepare
    self.prepare_url(url, params)
  File "/home/centos/myenv/lib64/python3.6/site-packages/requests/models.py", line 390, in prepare_url
    raise MissingSchema(error)
requests.exceptions.MissingSchema: Invalid URL '/login/logout.do': No schema supplied. Perhaps you meant http:///login/logout.do?
```

some url_prefix seems missing, and the rest of the function seems broken as well.

**Comment by mharista** [comment]
Hi @freedge 

I'll look into this and get the issue resolved.

**Comment by mharista** [comment]
Fixed abaf2577afb5b9b5e9d99a6b848ca2e987c22e66
